overlord/snapstate/backend/link_test.go

```diff
@@ -295,11 +295,11 @@ type: snapd
 `
 	snapdUnits := [][]string{
 		// system services
-		{"lib/systemd/system/snapd.service", "[Unit]\nExecStart=/usr/lib/snapd/snapd\n# X-Snapd-Snap: do-not-start"},
+		{"lib/systemd/system/snapd.service", "[Unit]\n[Service]\nExecStart=/usr/lib/snapd/snapd\n# X-Snapd-Snap: do-not-start"},
```

Comment:
the patch looks buggy
11:03 you have [Unit] and then immediately [Service]
11:03 in the past [Unit] had ExecStart
11:03 is that expected?

Reply:
As explained on IRC, this makes the test data look more like the actual service units that we ship, i.e. ExecStart is part of [Service] and there is a [Unit] section at the beginning of the file.